packages/xstate-graph/src/types.ts

```diff
@@ -135,7 +135,11 @@ export interface VisitedContext<TState, TEvent> {
 
 export interface SerializationOptions<TState, TEvent extends EventObject> {
   eventCases: EventCaseMap<TState, TEvent>;
-  serializeState: (state: TState, event: TEvent | null) => string;
+  serializeState: (
+    state: TState,
+    event: TEvent | undefined,
+    prevState?: TState
```

Comment:
So, we add `prevState` to the type for `serializeState` and see where the red squiggly lines are in `graph.ts`